README.md

```diff
@@ -1,47 +1,29 @@
 [![CircleCI](https://circleci.com/gh/Synthetixio/synthetix-exchange.svg?style=svg&circle-token=fd3dc3a4369c1e281e05262f695d59f94cf5340f)](https://circleci.com/gh/Synthetixio/synthetix-exchange) [![Netlify Status](https://api.netlify.com/api/v1/badges/36a95f47-4d7c-4fd2-be00-fc28fa5822e9/deploy-status)](https://app.netlify.com/sites/synthetix/deploys) [![Discord](https://img.shields.io/discord/413890591840272394.svg?color=768AD4&label=discord&logo=https%3A%2F%2Fdiscordapp.com%2Fassets%2F8c9701b98ad4372b58f13fd9f65f966e.svg)](https://discordapp.com/channels/413890591840272394/)
 [![Twitter Follow](https://img.shields.io/twitter/follow/synthetix_io.svg?label=synthetix_io&style=social)](https://twitter.com/synthetix_io)
 
-> Note: This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Synthetix.Exchange
+
+The code that powers [https://synthetix.exchange](https://synthetix.exchange).<br />
+It is powered by [synthetix-data](https://github.com/Synthetixio/synthetix-data) and [synthetix-js](https://github.com/Synthetixio/synthetix-js).
```

Comment:
powers, powered. Maybe rephrase?

Reply:
"The code for the Synthetix.Exchange dApp"?

Reply:
Yeah I'm good with that.